h2o-docs/src/product/performance-and-prediction.rst

```diff
@@ -2984,6 +2984,148 @@ For classification problems, when running ``h2o.predict()`` or ``.predict()``, t
 - If you train a model with training data and set the ``nfolds`` parameter, the Max F1 threshold from the training data model metrics is used.
 - If you train a model with the train data and validation data and also set the ``nfolds`` parameter, the Max F1 threshold from the validation data model metrics is used.
 
+Using the previous example, run the following code to find and interact with the model's threshold:
+
+.. tabs::
+    .. code-tab:: r R
+
+        # Retrieve your default threshold:
+        model@model$default_threshold
+        [1] 0.4393567
+
+        # Threshold is important to decide final prediction. Model prediction output 
+        # is a real number between 0-1 and the threshold is used to decide if the 
+        # output will be 0 or 1. Threshold is decided by max F1 metric by default, but
+        # you can reset the threshold:
+        new_threshold <- 0.6917189903
+        old_threshold <- h2o.reset_threshold(model, new_threshold)
+        reseted_model <- h2o.getModel(model@model_id)
+        # Predict with the reset model:
+        reset_pred <- h2o.predict(reseted_model, pros)
+        # Compare the predictions:
+        pred
+          predict        p0         p1
+        1       1 0.5332915 0.46670852
+        2       0 0.9199776 0.08002236
+        3       0 0.9170936 0.08290638
+        4       0 0.9519531 0.04804688
+        5       0 0.9512314 0.04876858
+        6       1 0.5405286 0.45947143
+
+        [73 rows x 3 columns] 
+        reset_pred
+          predict        p0         p1
+        1       0 0.9356172 0.06438281
```

Comment:
Did you use the same dataset? Because the `p0` and `p1` should be the same, only the `predict` value should be changed based on the new threshold.. :thinking:

Reply:
[Ohhhh, they do use a different dataset! And their models are built differently...](https://docs.h2o.ai/h2o/latest-stable/h2o-docs/performance-and-prediction.html#in-memory-prediction) Actually, I think this might be an issue...if you run the R example as it is on the page, you don't even get the correct printout that it says you should be getting. I think this example is just wrong. I think that this needs to be updated. Can I make a ticket so I can make these examples match?

For this PR though, I reran the R example using all the Python info: same dataset, same exact model build instead of following the examples on the page and this time I got the matching information.